runtime/onert/core/src/compiler/ManualScheduler.cc

```diff
@@ -29,22 +29,32 @@ namespace onert
 namespace compiler
 {
 
-ManualScheduler::ManualScheduler(const compiler::ManualSchedulerOptions &options)
-    : _options{options}
+ManualScheduler::ManualScheduler(const backend::BackendContexts &backend_contexts,
+                                 const compiler::CompilerOptions &options)
+    : _backend_contexts{backend_contexts}, _options{options}
 {
 }
 
 std::unique_ptr<BackendResolver> ManualScheduler::schedule(const ir::Graph &graph)
 {
+  const auto &manual_options = _options.manual_scheduler_options;
   auto backend_resolver = std::make_unique<compiler::BackendResolver>();
 
+  // This fallback will be used for unavailable backends
+  auto fallback = [&]() -> const backend::Backend * {
+    for (auto backend_id : _options.backend_list)
+    {
+      auto backend = resolveBackend(backend_id);
+      if (backend)
+        return backend;
+    }
+    return nullptr;
+  }();
+  assert(!fallback); // There must be at least one fallback
+
```

Comment:
@wateret Is it assume that all available backends are loaded before this method is called?

Reply:
@hseok-oh Yes. We have `BackendContexts`, that means we have all avaliable backends loaded.

FYI,

`BackendManager` manages dynamic libraries, this is a singleton and may contain backends that are not used for some sessions. It contains the whole set of backends that are used at least once in all the sessions of the current process.

`BackendContexts` is a set of backend contexts that are used for this session(compiler/execution). So `BackendContexts` is a subset(with context) of BackendManager.